Tasks/BashV3/task.json

```diff
@@ -67,7 +67,7 @@
             "label": "Script",
             "visibleRule": "targetType = inline",
             "required": true,
-            "defaultValue": "# Write your commands here",
+            "defaultValue": "# Write your commands here\n\n# Use the environment variables input below to pass secret variables to this script.",
```

Comment:
nit: The first sentence doesn't have a comment. The second sentence does. Consider making them consistent.

Reply:
Will do.